### PR TITLE
Add new 'customerBookings' root query

### PR DESCRIPTION
**Pull request by mrtnzlml**
Introducing new shiny query. This query replaces the old `allBookings` query as well as `booking` query (use `node` instead).

BTW the way how we work with `node` is incredibly painful...

### CODE REVIEW

File: src/booking/queries/Booking.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import type { BookingsItem } from '../Booking';
 export default {
   type: GraphQLBooking, // may be null (doesn't exist)
   description: 'Find booking by its id or databaseId.',
+  deprecationReason: 'Use "node" query instead.',
```
Comment (jaroslav-kubicek) [inline src/booking/queries/Booking.js]: I thought we don't like `node` query much 🙂

**Comment by michalsanger** [comment]
Do we have any naming strategy? I thought that query returning connection should be prefixed with `all`

**Comment by tbergq** [comment]
Why do we need to prefix all queries with `all`? is not the naming clarifying enough?

**Comment by michalsanger** [comment]
I was used to this:
 - prefix with `all` if query returns connection
 - use plural if query returns list

I agree it's no big help, just wanna make it clear if this convention is useful for us.

**Comment by tbergq** [comment]
I was taught that prefixing and suffixing is a bad naming practice. 

If every file is named like `MMBCityImage` and `MMBFlightList` and so on, it makes readability worse than just `CityImage` and `FlightList`

File: src/booking/queries/Booking.js (hunk 1 above)
Comment (mrtnzlml) [reply src/booking/queries/Booking.js]: We (I) don't like `node` query implementation much. Otherwise, it's brilliant.

**Comment by mrtnzlml** [comment]
No such a convention. Unless we have eslint rule for it (or something that will help with it automatically). It's also a little bit odd to ask for `allFlights(onlySomething: true)`...

**Comment by michalsanger** [comment]
OK, should we deprecate those `all` queries?

**Comment by mrtnzlml** [comment]
Just keep evolving. No need to spend hours on rewriting it now. Maybe we'll go for `allCustomerBookings` after we'll discover how I messed it up again.

**Comment by michalsanger** [comment]
I didn't mean right now :-) I just prefer naming consistency you know...

**Comment by mrtnzlml** [comment]
Yes, I know. However, there are big objections against this convention and I want to step out and try it differently. Let's see... :)

**Pull request merged**